docs/api-reference/regularization-l1-l2.md

```diff
@@ -0,0 +1,27 @@
+This class use [empricial risk minimization](https://en.wikipedia.org/wiki/Empirical_risk_minimization)
```

Comment:
Why is this in the l1-norm and l2-norm regularization include?

Reply:
We are optimize `regularized` ERM. `ER` is also known as loss function.

Reply:
I will add `Note that empricial risk is usually measured by applying a loss function on the model's predictions on collected data points.`

---
In reply to: [278778511](https://github.com/dotnet/machinelearning/pull/3586#discussion_r278778511) [](ancestors = 278778511)